services/api/handlers/projects.js

```diff
@@ -204,9 +205,15 @@ exports.get = {
 
 exports.patch = {
   update: function(request, reply) {
+    // hapi sets a pre-value to null if you pass reply an empty string :/
```

Comment:
I'm going to say no, lets keep '' as the default value for description so that we're not passing null values around.